From 6e2acd77a1a559b724aca2fa7a011a11dc4241c9 Mon Sep 17 00:00:00 2001
From: Jim Bride <jim.bride@intel.com>
Date: Fri, 3 Aug 2012 15:39:53 -0700
Subject: core/pathmap.mk: Add ethernet source

Change-Id: Id7ef6fcfafd36f1c288fa14f587a0d42d1996247
Depends-Change-Id: Ie1d2100e6324052057c267997d050b60761f009a
Depends-Change-Id: I049320d81fe510245c6d92f48eaf25ea7d7b908f
Depends-change-Id: I1766488371186391929b8dc6605cebf215c0baec
Depends-Change-Id: Id7ef6fcfafd36f1c288fa14f587a0d42d1996247
Depends-Change-Id: I39b79940e0495f915f602a16d99e1848bd3f2015
Signed-off-by: Matt Gumbel <matthew.k.gumbel@intel.com>
---
 core/pathmap.mk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/core/pathmap.mk b/core/pathmap.mk
index b76e0ed..5959c77 100644
--- a/core/pathmap.mk
+++ b/core/pathmap.mk
@@ -96,6 +96,7 @@ FRAMEWORKS_BASE_SUBDIRS := \
 	    telephony \
 	    wifi \
 	    keystore \
+	    ethernet\
 	 )
 
 #
-- 
1.7.4.1

